data-final-fixes.lua

```diff
@@ -26,7 +26,7 @@ data.raw.fish["fish"].pictures[2].draw_as_shadow = true
 data.raw.fish["fish"].selection_priority = 48
 
 -- Change technology requirement if oil is not available on land
-if settings.startup["deep_oil"].value and (settings.startup["no_oil_on_land"].value or settings.startup["no_oil_for_oil_rig"].value) then
+if settings.startup["deep_oil"].value and not mods["nullius"] and (settings.startup["no_oil_on_land"].value or settings.startup["no_oil_for_oil_rig"].value) then
```

Comment:
This should be handled by adding Nullius to the list of mods in `settings-updates.lua` which cause the oil setting to be forced to false.